models/perceptron.py

```diff
@@ -6,19 +6,16 @@
 import theano.tensor as T
 
 from smartpy import Model
-from utils import load_dict_from_json_file, save_dict_to_json_file
-from utils import WeightsInitializer
-
-
-dtype = theano.config.floatX
+from utils import load_dict_from_json_file, save_dict_to_json_file, WeightsInitializer
```

Comment:
Should we load `load_dict_from_json_file` and `save_dict_to_json_file` from smartpy's utils?